test/lib/core/loader/load_boot.test.js

```diff
@@ -14,7 +14,7 @@ describe('test/lib/core/loader/load_boot.test.js', () => {
   });
 
   it('should load app.js', async () => {
-    app.mockLog();
+    app.mockLog(app.coreLogger);
```

Comment:
thanks, it's really a bug, but should not modify here.

the problem is  lifecyle, egg-mock (https://github.com/eggjs/egg-mock/blob/master/lib/app.js#L48) delay the lifecyle, so `app.mockLog` is exec after egg-ready.

this place should just remove `app.mockLog` which will let `app.expectLog` check the file at disk.

Reply:
```suggestion

```